docs/mvp_morevp_interaction.md

```diff
@@ -86,3 +97,5 @@ A: I don't see a way to do it yet.
 Q: When IFE/piggyback on output is added, what is the cost of checking if standard exits were performed from output(s)?  
 A: Number of outputs is known on both IFE and piggyback; single SLOAD per output?  
 
+Q: What about mapping between IFE and SE from a deposit?
```

Comment:
I think it is not clear what the link between the question and answer here. I prefer the way you put in issue:
eg.
Q: Is there any issue/concern from having different exit id for deposit tx and regular tx?
A: The only problem is the fact that if we started an IFE from a deposit, we could not check whether someone tried to also SE the deposit. But this is not a problem since we cannot start an IFE from the deposit: `sum of inputs < sum of outputs`.